src/commands/app/delete/ci.js

```diff
@@ -10,26 +10,41 @@ governing permissions and limitations under the License.
 */
 
 const BaseCommand = require('../../../BaseCommand')
-const yeoman = require('yeoman-environment')
+const path = require('path')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app:add:action', { provider: 'debug' })
 const { Flags } = require('@oclif/core')
-const generators = require('@adobe/generator-aio-app')
```

Comment:
Can the `delete-ci` generator now be removed from this lib?

Reply:
yes, especially since it is now bumped to a major version